frontend/src/__tests__/integration/home-page.test.tsx

```diff
@@ -0,0 +1,131 @@
+import HomePage from '@/pages/home-page';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { BrowserRouter } from 'react-router-dom';
+
+const mockedUseNavigate = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...(jest.requireActual('react-router-dom') as Record<string, any>),
+  useNavigate: () => mockedUseNavigate,
+}));
+
+afterEach(() => mockedUseNavigate.mockRestore());
+
+describe('Integration Test: Home Route', () => {
+  test('Home Route: renders home page', async () => {
+    //ARRANGE
```

Comment:
Can we rename this to `Home Route: Renders home page`, Caps after colon.... Just to follow the some kind of uniformity, In every test case title